ui/packages/catalog-ui-search/src/main/webapp/react-component/location/bounding-box.js

```diff
@@ -74,34 +93,129 @@ const BoundingBoxLatLon = props => {
       />
       <TextField
         label="North"
-        value={north}
-        onChange={cursor('north')}
+        value={north !== undefined ? String(north) : north}
+        onChange={value => onChangeDd('north', value)}
+        onBlur={() => setDdError(validateGeo('north', north))}
         type="number"
         step="any"
         min={northMin || -90}
         max={90}
```

Comment:
I know this was existing code, but just wondering why we would have a max for `north` and not `south`, and no max at all for `east` and `west`... weird. Maybe we should make it consistent?